docs/app/pages/components/components-sections/select/select-custom/select-custom.component.html

```diff
@@ -0,0 +1,134 @@
+            [busy]="showBusyIndicator"
+            [maxHeight]="componentMaxHeight"
+            [placeholder]="placeholder">
+
+            <ng-template #buttonContent>
+                <b>Selection:</b> {{ selected ? selected.name : '(none)' }}
+            </ng-template>
+
+            <ng-template #dropdownContent>
+
+                <div class="radio-button-container"
+                     uxTabbableList>
+                    <ux-radio-button
+                        uxTabbableListItem
+                        *ngFor="let option of filteredOptionList"
+                        name="group"
+                        [value]="selected"
+                        (valueChange)="selected = $event"
+                        [option]="option"
+                        (keydown.space)="selectButton($event, option)">
+                        <span [innerHTML]="option.name | highlightSearch: filter">
+                        </span>
+                    </ux-radio-button>
+                </div>
+            </ng-template>
+
+        </ux-select-custom>
+    </div>
+</div>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <ux-accordion>
+            <ux-accordion-panel class="accordion-chevron" heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                    <div class="col-md-2 col-sm-12">
+                        <label>maxHeight</label>
+                    </div>
+                    <div class="col-md-2 col-sm-12">
+                        <ux-number-picker-inline [(value)]="componentMaxHeight"></ux-number-picker-inline>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="allowNull">allowNull</ux-checkbox>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="showBusyIndicator">busy</ux-checkbox>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-2 col-sm-12">
+                        <label>placeholder</label>
+                    </div>
+                    <div class="col-md-2 col-sm-12">
+                        <input class="form-control" [(ngModel)]="placeholder">
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <button class="btn button-primary" (click)="selected = filteredOptionList[0]">Select first element</button>
+                    </div>
+                </div>
+            </ux-accordion-panel>
+        </ux-accordion>
+    </div>
+</div>
+
+<hr>
+
+<p>
+    The <code>select-custom</code> component is a drop-down with flexible content.
```

Comment:
`ux-input-dropdown` was the agreed name.

Reply:
Changed with the renaming.